<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EchoCloseHint</class>
 <widget class="QWidget" name="EchoCloseHint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetBack" native="true">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widgetBack
{
border-radius:5px;
border:1px solid #cccccc;
background-color: rgba(255, 255, 255, 0);
border-image: url(:/images/msgBox_bk.png);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_back">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>33</number>
      </property>
      <property name="topMargin">
       <number>35</number>
      </property>
      <property name="rightMargin">
       <number>33</number>
      </property>
      <property name="bottomMargin">
       <number>35</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetTittle" native="true">
        <property name="minimumSize">
         <size>
          <width>357</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>357</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widgetTittle
{
border-top-left-radius: 2px;
border-top-right-radius: 2px;
}
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelTitle">
           <property name="minimumSize">
            <size>
             <width>272</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>关闭提示</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClose">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
border-image: url(:/images/close_play1.png);
background-repeat:no-repeat;
border:none;
}
QPushButton:hover
{
border-image:url(:/images/close_play2.png);
}
QPushButton:pressed
{
border-image:url(:/images/close_play3.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_msg">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="leftMargin">
           <number>40</number>
          </property>
          <property name="rightMargin">
           <number>30</number>
          </property>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>258</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>258</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:#999999;</string>
            </property>
            <property name="text">
             <string>点击关闭按钮时：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonHide">
            <property name="minimumSize">
             <size>
              <width>258</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>258</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:#666666;
text-align:left;</string>
            </property>
            <property name="text">
             <string>最小化到系统托盘</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonClose">
            <property name="minimumSize">
             <size>
              <width>258</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>258</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:#666666;
text-align:left;</string>
            </property>
            <property name="text">
             <string>退出echo客户端</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>55</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>355</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>355</width>
            <height>42</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#widget
{
border-bottom-left-radius: 5px;
border-bottom-right-radius: 5px;
background-color: rgb(247, 247, 247);
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_button">
             <property name="leftMargin">
              <number>30</number>
             </property>
             <property name="rightMargin">
              <number>30</number>
             </property>
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox
{
color:#999999;
text-align:left;
}
QCheckBox:hover
{
color:#666666;
}
</string>
               </property>
               <property name="text">
                <string>不再提醒</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>28</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background:#6ed56c;
	border-radius:5px;
	color:#ffffff;
}
QPushButton:hover{
    background-color:#6ee56c
}
QPushButton:pressed{
    background-color:#6ee56c
}</string>
               </property>
               <property name="text">
                <string>确认</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
